src/components/graph/menus/model-node-editor.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React, { useState } from 'react';
+import { NativeSelect, TextField } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import { useIntl } from 'react-intl';
+
+const ModelNodeEditor = ({ selectedNode }) => {
+    const intl = useIntl();
+
+    const [nameValue, setNameValue] = useState(selectedNode.data?.label);
+    const [modelType, setModelType] = useState(selectedNode.modelType);
+
+    const updateNameValue = (event) => {
+        setNameValue(event.target.value);
+    };
+
+    const updateModelType = (event) => {
+        setModelType(event.target.value);
+    };
+
+    const handleValidate = (event) => {
+        selectedNode.data.label = nameValue;
```

Comment:
Do we need to call updateTreeNode rest API here ? like for NetworkModificationNode

Reply:
Yes, I forgot to call it here.